linux/debian/jamulus-headless.postinst

```diff
@@ -6,4 +6,7 @@ set -e
 
 adduser --system --quiet --home /nonexistent --no-create-home jamulus
 
+sudo mkdir /etc/jamulus
```

Comment:
This will fail if the directory already exists.  It will also 
```suggestion
sudo mkdir -p /etc/jamulus
```

Reply:
Why is sudo required anyway? If adduser I'd executable meditate should also work without sudo. Nevertheless, I believe there's another way to add files and folders which is then also automatically cleaned up by dpkg/apt. Check out deploy_linux.sh

Reply:
Writing in /etc might require root access, but I can't explain why adduser doesn't in this instance.

There is no deploy_linux.sh. Do you mean deploy_deb.sh ?

If I am uninstalling an application, there's a case for intentionally not removing a small file the user has customized.

Are there examples of our files that are handled in the auto-cleanup way you prefer?